### DISCUSSION TITLE
Feature Request - Conversion-Pause capability

### DISCUSSION THREAD

**Discussion by b1ueliner** [openaudible/openaudible#1257] [Q&A]
It would be nice to have a way to hit pause on the conversion process if that is possible. When I am converting a large number of audiobooks (to MP3) it really smacks my CPU and incapacitates my machine to the point where it is basically unusable while conversions are happening. It looks like 3 or 4 books are being converted at the same time. Maybe, it would be good to have a way to set the number of simultaneous conversions to one at a time or some custom selectable number?

Love the product,

mike

**Comment by openaudible** [comment]
Simultaneous connections can be edited in the settings.json file in the openaudible prefs directory (visible/accessible in the prefs dialog).

Edit settings.json and update the following:

```
  "concurrentConversions": 3,
  "concurrentDownloads": 3,
```

Currently there isn't a way to pause -- but you can select all, cancel all jobs (or simply quit) and those jobs can be resumed when you restart. (Not elegant, but pausing isn't a common request.)
